app/models/user.rb

```diff
@@ -47,7 +45,7 @@
 #
 class User < ApplicationRecord
   include PgSearch::Model
-
+  self.ignored_columns = ["encrypted_access_token", "encrypted_access_token_iv"]
```

Comment:
may want to record in a commit message that this is being removed because it's an ancient zendesk thing